project_templates/technote_aastex/{{cookiecutter.series.lower()}}-{{cookiecutter.serial_number}}/.travis.yml

```diff
@@ -0,0 +1,14 @@
+sudo: true
+dist: xenial
+services:
+  - docker
+language: python
+python:
+  - '3.7'
+before_install:
+  - "pip install 'lander>=0.1.0,<0.2'"
+script:
+  # Compile PDF using containerized lsst-texmf
+  - "docker run --rm -v `pwd`:/workspace -w /workspace lsstsqre/lsst-texmf:latest sh -c 'make'"
+  # Deploy website. See https://github.com/lsst-sqre/lander for CLI options
+  - "lander --upload --pdf {{ cookiecutter.series.upper() }}-{{ cookiecutter.serial_number }}.pdf --lsstdoc {{ cookiecutter.series.upper() }}-{{ cookiecutter.serial_number }}.tex --env=travis --ltd-product {{ cookiecutter.series.lower() }}-{{ cookiecutter.serial_number }}"
```

Comment:
Maybe `--lsstdoc` will still work with AASTeX. It'll probably grab at least the title and abstract fields, which is useful. No need to change this yet.

Reply:
yes that was the idea